python/pyspark/ml/clustering.py

```diff
@@ -58,6 +58,124 @@ def gaussiansDF(self):
         """
         return self._call_java("gaussiansDF")
 
+    @property
+    @since("2.0.0")
+    def summary(self):
+        """
+        Gets summary of model on training set. An exception is thrown if
+        `trainingSummary is None`.
```

Comment:
this also seems strangely formatted - did you mean `trainingSummary` is `None`. ?